Bio/PDB/parse_pdb_header.py

```diff
@@ -333,6 +354,13 @@ def _parse_pdb_header_list(header):
                 pdbh_dict["author"] += auth
             else:
                 pdbh_dict["author"] = auth
+        # parse SSBOND records and add them to the Header dict
+        elif key == "SSBOND":
```

Comment:
Why not do something like in lines `259-261` above and use the `_get_ssbond` method to directly grab the ss bond info from the header? That would make the function much easier to test (a single call) and you can do all the manipulation you need inside of it.

In other words:

```python
pdbh_dict["ssbond"] = _get_ssbond(header)
```